diff --git a/base/dist/emotion-styled-base.browser.development.esm.js b/base/dist/emotion-styled-base.browser.development.esm.js
index a7dea519384d31445a9bcbe3e3986904cf46ede8..f761e25463f66fed0cc6369b86dcba0890fa56df 100644
--- a/base/dist/emotion-styled-base.browser.development.esm.js
+++ b/base/dist/emotion-styled-base.browser.development.esm.js
@@ -148,6 +148,7 @@ var createStyled = function createStyled(tag, options) {
       if (ref) {
         newProps.ref = ref;
       }
+      performance.mark('Emotion_start');
 
       return /*#__PURE__*/React.createElement(React.Fragment, null, /*#__PURE__*/React.createElement(Insertion, {
         cache: cache,
diff --git a/base/dist/emotion-styled-base.browser.esm.js b/base/dist/emotion-styled-base.browser.esm.js
index 7c467c61933840922b75985607cae61ad245b0dd..d082fb4b2d126241fb9fee995bbd14d0b81b403d 100644
--- a/base/dist/emotion-styled-base.browser.esm.js
+++ b/base/dist/emotion-styled-base.browser.esm.js
@@ -134,6 +134,7 @@ var createStyled = function createStyled(tag, options) {
       if (ref) {
         newProps.ref = ref;
       }
+      performance.mark('Emotion_start');
 
       return /*#__PURE__*/React.createElement(React.Fragment, null, /*#__PURE__*/React.createElement(Insertion, {
         cache: cache,
