diff --git a/cjs/react-dom-client.development.js b/cjs/react-dom-client.development.js
index 83ea1c6a783efe2e674eda4b681eb361f000d12d..c84fe7324cba0e23d0ecc13cfb7cbe534ef1619e 100644
--- a/cjs/react-dom-client.development.js
+++ b/cjs/react-dom-client.development.js
@@ -19740,6 +19740,9 @@
         ? prior.parentNode.insertBefore(instance, prior.nextSibling)
         : ((precedence = 9 === root.nodeType ? root.head : root),
           precedence.insertBefore(instance, precedence.firstChild));
+      performance.mark('Style_end', {
+        detail: instance.dataset.href
+      });
     }
     function adoptPreloadPropsForStylesheet(stylesheetProps, preloadProps) {
       null == stylesheetProps.crossOrigin &&
diff --git a/cjs/react-dom-client.production.js b/cjs/react-dom-client.production.js
index af7283abaca3e5812ef71b03d9c999410baa1a80..50d3aa868663022dc51d7074b4f332b772fee27c 100644
--- a/cjs/react-dom-client.production.js
+++ b/cjs/react-dom-client.production.js
@@ -14037,6 +14037,7 @@ function acquireResource(hoistableRoot, resource, props) {
       insertStylesheet(instance, props.precedence, hoistableRoot));
   return resource.instance;
 }
+
 function insertStylesheet(instance, precedence, root) {
   for (
     var nodes = root.querySelectorAll(
@@ -14056,6 +14057,9 @@ function insertStylesheet(instance, precedence, root) {
     ? prior.parentNode.insertBefore(instance, prior.nextSibling)
     : ((precedence = 9 === root.nodeType ? root.head : root),
       precedence.insertBefore(instance, precedence.firstChild));
+      performance.mark('Style_end', {
+        detail: instance.dataset.href
+      });
 }
 function adoptPreloadPropsForStylesheet(stylesheetProps, preloadProps) {
   null == stylesheetProps.crossOrigin &&
